### DISCUSSION TITLE
Lack of addons

### DISCUSSION THREAD

**Discussion by davidjeba** [statamic/cms#5672] [General]
statamic doesnt have addons to create a website like 
[airbnb.com](https://www.airbnb.com/)
[neighbor.com](https://www.neighbor.com/)

**Comment by duncanmcclean** [comment]
What sort of addons do you need? 

If I was to build something like that, I'd probably write some custom site code anyways. 

Feel free to create some.

**Reply by davidjeba** [reply]
Wish i could, but im a product guy... not a coder.
how i see is, if statamic can release a major module for bookings related commerce
it could add huge advantage like how woocommerce has become a major advantage for wordpress

but again, its not worth it as its based on PHP
if statamic is built on django there could have been a tremendous response from developers

**Comment by afonic** [comment]
We have actually made one but it doesn't have and docs yet: https://github.com/reachweb/statamic-resrv

You can find me in Discord and I can help you until we're done with the documentation and starter packs.

**Reply by davidjeba** [reply]
Wow, this is perfect start.
ill look into your package
